pkg/github/helper_test.go

```diff
@@ -10,6 +10,15 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// expectPath is a helper function to create a partial mock that expects a
+// request with the given path, with the ability to chain a response handler.
+func expectPath(t *testing.T, expectedPath string) *partialMock {
```

Comment:
I like expect path. I think it will catch both errors matching args into API requests and protect from regressions also.

GitHub API is very stable and I don't expect much maintenance will be required.